Sources/PackageModelSyntax/AddTarget.swift

```diff
@@ -57,6 +57,62 @@ public enum AddTarget {
         }
     }
 
+    // Check if the package has a single target with that target's sources located
+    // directly in `./Sources`. If so, move the sources into a folder named after
+    // the target before adding a new target.
+    public static func moveSingleTargetSources(
```

Comment:
Is this supposed to be used anywhere outside of this package? If not, we should prefer `package` or lower visibility by default.
```suggestion
    package static func moveSingleTargetSources(
```